docs/git-guide.md

```diff
@@ -1357,6 +1356,46 @@ HEAD is now at 99aa2bf Add provision.py fails issue in common erros
 Current branch review-1913 is up to date.
 ```
 
+### Fetch a pull request without rebasing
+
+`tools/fetch-pull-request` is a similar to `tools/fetch-rebase-pull-request`, but
+it does not rebase the pull request against upstream/master, thereby getting
+exactly the same repository state as the commit author had.
+
+Run the script with the ID number of the pull request as the first argument.
+
+```
+$ tools/fetch-pull-request 5156
++ git diff-index --quiet HEAD
++ request_id=5156
++ remote=upstream
++ git fetch upstream pull/5156/head
+From https://github.com/zulip/zulip
+ * branch            refs/pull/5156/head -> FETCH_HEAD
++ git checkout -B review-original-5156
+Switched to a new branch 'review-original-5156'
++ git reset --hard FETCH_HEAD
+HEAD is now at 5a1e982 tools: Update clean-branches to clean review branches.
+```
+
+### Delete unimportant branches
+
+`tools/clean-branches` is a shell script that removes branches that are either:
+
+1. Local branches ancestors of origin/master
+2. Branches in origin ancestors of origin/master and named like `$USER-*`.
```

Comment:
Same thing, please consider replacing with `Branches in origin which are ancestors of origin/master`. Without a connector such as `which are` or even `that are`, it seems ambiguous. :)